Makefile

```diff
@@ -14,10 +14,10 @@
 
 # build binary
 static:
-	go build -o aws-k8s-agent main.go
-	go build -o aws-cni plugins/routed-eni/cni.go
-	go build verify-aws.go
-	go build verify-network.go
+	GOOS=linux CGO_ENABLED=0 go build -o aws-k8s-agent main.go
+	GOOS=linux CGO_ENABLED=0 go build -o aws-cni plugins/routed-eni/cni.go
+	go build scripts/verify-aws.go
```

Comment:
Can we put them in scripts/verify-aws/verify-aws.go then go install will work correctly.